prime-router/src/main/kotlin/cli/ProcessFhirCommands.kt

```diff
@@ -151,65 +157,159 @@ class FhirPathCommand : CliktCommand(
         val contents = inputFile.inputStream().readBytes().toString(Charsets.UTF_8)
         if (contents.isBlank()) throw CliktError("File ${inputFile.absolutePath} is empty.")
         val bundle = FhirTranscoder.decode(contents)
+        focusResource = bundle
+
         echo("", true)
         echo("Using the FHIR bundle in ${inputFile.absolutePath}...", true)
+        echo("Special commands:", true)
+        echo("  resource [= | :] <FHIR Path>   - Sets %resource to a given FHIR path", true)
         echo("Press CTRL-C or ENTER to exit.", true)
 
         // Loop until you press CTRL-C or ENTER at the prompt.
         while (true) {
+            echo("", true)
+            echo("%resource = $focusPath")
```

Comment:
This was really helpful in not getting confused about what I had already set as the resource